examples/chicane/run_chicane.py

```diff
@@ -48,17 +48,18 @@
 
 # design the accelerator lattice
 ns = 25  # number of slices per ds in the element
-rc = 10.35  # bend radius (meters)
+rc = 10.3462283686195526  # bend radius (meters)
 psi = 0.048345620280243  # pole face rotation angle (radians)
+lb = 0.500194828041958  # projected bend length (meters)
```

Comment:
The comment says `projected bend length` but it is the arc length, no? The projected length would be the exact 0.5 m.